apps/docs/src/content/reference/gltf/getting-started.mdx

```diff
@@ -310,4 +330,163 @@ If you want to use the component [`<Suspense>`](/docs/reference/extras/suspense)
+    if (existsSync(componentPath)) {
+      console.error(`File ${componentPath} already exists, skipping.`)
+      return false
+    }
+  }
+  return true
+})
+
+if (filteredGltfFiles.length === 0) {
+  console.log('No gltf or glb files to process.')
+  exit()
+}
+
+filteredGltfFiles.forEach((file) => {
+  // run the gltf transform command on every file
+  const path = join(configuration.sourceDir, file)
+
+  // parse the configuration
+  const args: string[] = []
+  if (configuration.root) args.push(`--root ${configuration.root}`)
+  if (configuration.types) args.push('--types')
+  if (configuration.keepnames) args.push('--keepnames')
+  if (configuration.meta) args.push('--meta')
+  if (configuration.shadows) args.push('--shadows')
+  args.push(`--printwidth ${configuration.printwidth}`)
+  args.push(`--precision ${configuration.precision}`)
+  if (configuration.draco) args.push(`--draco ${configuration.draco}`)
+  if (configuration.preload) args.push('--preload')
+  if (configuration.suspense) args.push('--suspense')
+  if (configuration.isolated) args.push('--isolated')
+  if (configuration.transform.enabled) {
+    args.push(`--transform`)
+    args.push(`--resolution ${configuration.transform.resolution}`)
+    if (configuration.transform.simplify.enabled) {
+      args.push(`--simplify`)
+      args.push(`--weld ${configuration.transform.simplify.weld}`)
+      args.push(`--ratio ${configuration.transform.simplify.ratio}`)
+      args.push(`--error ${configuration.transform.simplify.error}`)
+    }
+  }
+  const formattedArgs = args.join(' ')
+
+  // run the command
+  const cmd = `npx @threlte/gltf@next ${path} ${formattedArgs}`
+  try {
+    execSync(cmd, {
+      cwd: configuration.sourceDir
+    })
+  } catch (error) {
+    console.error(`Error transforming model: ${error}`)
+  }
+})
+
+// read dir again, but search for .svelte files only.
+const svelteFiles = readdirSync(configuration.sourceDir).filter((file) => file.endsWith('.svelte'))
+
+svelteFiles.forEach((file) => {
+  // now move every file to /src/components/models
+  const path = join(configuration.sourceDir, file)
+  const newPath = join(configuration.targetDir, file)
+  copyFile: try {
+    // Sanity check, we checked earlier if the file exists. Still, the CLI takes
+    // a while, so who knows what happens in the meantime.
+    if (!configuration.overwrite) {
+      // check if file already exists
+      if (existsSync(newPath)) {
+        console.error(`File ${newPath} already exists, skipping.`)
+        break copyFile
+      }
+    }
+    copyFileSync(path, newPath)
+  } catch (error) {
+    console.error(`Error copying file: ${error}`)
+  }
+
+  // remove the file from /static/models
+  try {
+    unlinkSync(path)
+  } catch (error) {
+    console.error(`Error removing file: ${error}`)
+  }
+})
+```
+
+Place this script in `scripts/transform-models.ts` and run it with `npx tsx scripts/model-pipeline.ts`.
```

Comment:
could possibly provide a link to tsx, I didn't know about this package until now 😅